src/TimeSlice.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { PropTypes, Component } from 'react'
+import cn from 'classnames'
+
+export default class TimeSlice extends Component {
+  static propTypes = {
+    value: PropTypes.instanceOf(Date).isRequired,
+    isNow: PropTypes.bool,
+    showlabel: PropTypes.bool,
+    selected: PropTypes.bool,
+    content: PropTypes.string,
+    culture: PropTypes.string
+  }
+
+  static defaultProps = {
+    isNow: false,
+    showlabel: true,
+    selected: false,
+    culture: 'en',
```

Comment:
I wouldn't set a default here, in general we want the "default" to be the globally set culture...I realize that's leaky sounding from a component purity standpoint but its managed by the localizer and opaque to the component and or whatever reason that i18n libs decided enables the best interop :/